controls/include/controls/PID.h

```diff
@@ -0,0 +1,38 @@
+//
+// Created by tanvi on 10/30/2020.
+//
+
+#ifndef CONTROLS_PID_H

```

Comment:
Replace these old-style include guards with a single `#pragma once`

This replaces the entire include guard, including the `#endif` at the end of the file.